skimage/filters/rank/tests/test_rank.py

```diff
@@ -78,14 +85,12 @@ def setup(self):
         # Set again the seed for the other tests.
         np.random.seed(0)
         self.selem = morphology.disk(1)
-        self.refs = np.load(os.path.join(skimage.data_dir,
-                                         "rank_filter_tests.npz"))
```

Comment:
I think it would be less intrusive here to use `np.load(fetch(` here and forget about the pytest fixture. I like fixtures, but they seem unnecessary when using a UnitTest-style class.